### PR TITLE
Feature/errorbox

### PR DESCRIPTION
**Pull request by zaanposni**
I also fixed some intendation issues, thats why github shows every line as changed 🤣 

Preview:
grafik

### CODE REVIEW

**Comment by zaanposni** [comment]
Preview for online-status
Tested in
- Chrome
- Firefox
- Edge

grafik

**Comment by Monkmitrad** [comment]
Looks good. Probs for design

**Comment by SFFan123** [comment]
damm Bro!

File: src/app/routes.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ def auth_error():
 
 @auth.verify_token
 def verify_token(token):
+    return True
```
Comment (zaanposni) [inline src/app/routes.py]: upsi 🙃

**Pull request merged**